tasks/shell_crmsh/crm-cluster-properties.yml

```
@@ -1,15 +1,48 @@
 # SPDX-License-Identifier: MIT
 ---
+# stonith-enabled=true is required for clusters, but this task
+# allows for creation of cluster without stonith enabled.
+# It also ensures that stonith-enabled is always present.
+- name: Append stonith-enabled to ha_cluster_cluster_properties
+  ansible.builtin.set_fact:
+    __ha_cluster_property_attr: "{{
+      (properties_set.attrs + [{'name': 'stonith-enabled', 'value': 'true'}])
+      if properties_set.attrs | selectattr('name','equalto','stonith-enabled')
```

```suggestion
      if properties_set.attrs | selectattr('name', 'equalto', 'stonith-enabled')
```
To resolve a warning from ansible-lint

---

@tomjelinek It would be much easier if we raised 80 char limit ... because it becomes difficult to fit all of it while making it jinja2 compatible and readable at same time.

---

completed in latest commit